disconnect-when-backgrounded.js

```diff
@@ -1,18 +1,30 @@
 var disconnectTimer = null;
 
+// 10 seconds by default
+var disconnectTime = (Meteor.settings && Meteor.settings.public && Meteor.settings.public.disconnectTime || 10) * 1000;
```

Comment:
I mean that `Meteor.settings.public.disconnectTime` is only read on app bootup. If my app had the code:

```
Meteor.startup(() => {
   Meteor.settings.public.disconnectTime = 60;
});
```

Then the new disconnectTime wouldn't take effect because it's already been set. A better approach is to add it to move this line inside the `disconnectIfHidden` function - so it reads the new setting each time.